docs/Table.md

```diff
@@ -70,88 +95,155 @@ const { query } = initialize({ iotsitewiseClient });
 
 // jsx
 <iot-table
-  viewport={{ duration: '5m' }}
+  viewport = {{ duration: '5m' }}
   
-  tableColumns={[
+  items = {[
     {
-      header: 'Tempature',
-      rows: [
-        toId({ assetId: 'engine-1', propertyId: 'tempature' }),
-        toId({ assetId: 'engine-2', propertyId: 'tempature' }),
-      ]
+      rpm: {
+        $cellRef: {
+          id: toId({ assetId: 'turbine-1', propertyId: 'rotation_per_minute' }),
+          resolution: 0
+        }
+      },
+      wind_speed: {
+        $cellRef: {
+          id: toId({ assetId: 'turbine-1', propertyId: 'wind_speed' }),
+          resolution: 0
+        }
+      }
     },
     {
-      header: 'RPM',
-      rows: [
-        toId({ assetId: 'engine-1', propertyId: 'rpm' }),
-        toId({ assetId: 'engine-2', propertyId: 'rpm' }),
-      ]
+      rpm: {
+        $cellRef: {
+          id: toId({ assetId: 'turbine-2', propertyId: 'rotation_per_minute' }),
+          resolution: 0
+        }
+      },
+      wind_speed: {
+        $cellRef: {
+          id: toId({ assetId: 'turbine-2', propertyId: 'wind_speed' }),
+          resolution: 0
+        }
+      }
     }
   ]}
   
-  queries={[
+  columnDefinitions = {[
+    {
+      key: 'rpm',
+      header: 'RPM'
+    },
+    {
+      key: 'wind_speed',
+      header: 'Wind Speed'
+    }
+  ]}
+  
+  queries = {[
     query.timeSeriesData({
       assets: [
         { 
-          assetId: 'engine-1', 
+          assetId: 'turbine-1', 
           properties: [
-            { propertyId: 'tempature' },
-            { propertyId: 'rpm' }
+            { propertyId: 'rotation_per_minute' },
+            { propertyId: 'wind_speed' }
           ],
-        }, { 
-          assetId: 'engine-2', 
+        }, 
+        { 
+          assetId: 'turbine-2', 
           properties: [
-            { propertyId: 'tempature' },
-            { propertyId: 'rpm' }
-          ],
+            { propertyId: 'rotation_per_minute' },
+            { propertyId: 'wind_speed' }
+          ]
         }
-    ])
-  ]}}
+      ]
+    })
+  ]}
 />
 ```
 
 ## Properties
 
-The table component contains the following properties that you can customize. 
+The table component contains the following customizable properties:
 
-### `tableColumns`
+Note: Since the table component built over the `@awsui/components-react` table component, please refer to [AWS-UI/table](https://github.com/aws/awsui-documentation/blob/main/components/table.md) for most of the available properties. Differences in properties are listed below.
+
+### `items`
```

Comment:
need more information on `$cellRefs`, what they are, what they are used for, how the data becomes accesible and displayed on the table

Reply:
added a section for $cellRef